default.nix

```diff
@@ -16,7 +16,11 @@ let
   fetchTree =
     { info, locked, ... }:
     if locked.type == "github" then
-      { outPath = fetchTarball "https://api.github.com/repos/${locked.owner}/${locked.repo}/tarball/${locked.rev}";
+      {
+        outPath = fetchTarball {
+          url = "https://api.github.com/repos/${locked.owner}/${locked.repo}/tarball/${locked.rev}";
+          sha256 = info.narHash;
```

Comment:
Do you know what version of Nix introduced this hash format? I'd really like to have it, but might be better to have it optionally based on that?

Reply:
Good question. SRI hashes were introduced in Nix 2.2 according to https://github.com/NixOS/nix/commit/6024dc1d97212130c19d3ff5ce6b1d102837eee6

Reply:
Yeah, I think this is fine. We also just increased the minimum Nix version required by Nixpkgs to 2.2.